src/platform/packages/shared/response-ops/alerts-filters-form/components/alerts_filters_form.tsx

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React, { useState, useCallback, useEffect } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  useEuiTheme,
+  EuiPanel,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import useDeepCompareEffect from 'react-use/lib/useDeepCompareEffect';
+import {
+  AlertsFiltersExpression,
+  AlertsFiltersFormItemType,
+  AlertsFilter,
+  FlattenedExpressionItem,
+} from '../types';
+import { DELETE_OPERAND_LABEL } from '../translations';
+import { AlertsFiltersFormItem } from './alerts_filters_form_item';
+import {
+  flattenFiltersExpression,
+  reconstructFiltersExpression,
+} from '../utils/filters_expression';
+import {
+  ADD_AND_OPERATION_BUTTON_SUBJ,
+  ADD_OR_OPERATION_BUTTON_SUBJ,
+  DELETE_OPERAND_BUTTON_SUBJ,
+} from '../constants';
+
+export interface AlertsFiltersFormProps {
+  value?: AlertsFiltersExpression;
+  onChange: (newValue: AlertsFiltersExpression) => void;
+  isDisabled?: boolean;
+}
+
+const DEFAULT_VALUE: AlertsFiltersExpression = {
```

Comment:
This ensures that the form is initialized with an initially empty "Filter by" selector